js/Features.js

```diff
@@ -46,6 +46,13 @@ var Features = function (config) {
         );
     }
 
+    if (semver.gte(config.apiVersion, "1.21.0")) {
+        features.push(
+            {bit: 25, group: 'rxMode', mode: 'group', name: 'RX_SPI', haveTip: true},
```

Comment:
This is not really a configurator problem. The firmware treats these mutually exclusive settings as part of the feature set. The configurator actually tries to fix this by preventing users from selecting mutually exclusive settings like RX_PPM and RX_SERIAL at the same time.